src/CronTask.php

```diff
@@ -854,9 +854,12 @@ public static function launch($mode, $max = 1, $name = '')
                     $i
                 );
                 if ($crontask->getNeedToRun($mode, $name)) {
-                     $_SESSION["glpicronuserrunning"] = "cron_" . $crontask->fields['name'];
+                    $_SESSION["glpicronuserrunning"] = "cron_" . $crontask->fields['name'];
+                    $_SESSION["glpiactive_entity"]   = 0;
```

Comment:
```suggestion
                    Session::loadEntity(0, true);
```

When `cron` is likely to send notifications (`mailgate` then `new ticket` notification)

Notification filters are limited to the root entity

When GLPI try know if related item match filter

https://github.com/glpi-project/glpi/blob/50a87ef47b26febc1941d241ac3d7d572d34a4e5/src/Glpi/Search/FilterableTrait.php#L89-L91

Method use `getEntitiesRestrictRequest` that add `WHERE` clause on `entities_id` depending of `$_SESSION['glpiactiveentities_string'];`

the use of `Session::loadEntity(0, true);` allows you to fill missing data to work properly.